src/components/pages/Home/Home.tsx

```diff
@@ -75,8 +88,17 @@ function Home() {
       ) : (
         <>
           <Workflow
-            stage={stage}
+            stage={stage.toString()}
             onNext={() => {
+              if (stage == 1) {
+                const searchParameters = createSearchParams({
+                  data: [domain!, antID!.toString(), '1', '1'],
```

Comment:
this likely doesn't work - you cannot set an array of data in the URL, they'll all have a key of `data` so it's impossible to parse

Reply:
resolved in recent commits